src/style.css

```diff
@@ -0,0 +1,83 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+  font-weight: 400;
+}
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+.formBig {
+  margin-bottom: 20px;
+  margin-top: 20px;
+}
+
+.formSmall {
+  margin-bottom: 20px;
+}
+
+.bigSearch {
+  box-sizing: border-box;
+  width: 100%;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-weight: 300;
+  font-size: 16px;
+  height: 70px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 26px 25px;
+  padding-left: 62px;
+}
+
+.bigSearch:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  font-weight: 400;
+}
```

Comment:
this styles the same for big and small search bar, move it to one class and use it on both of them